### DISCUSSION TITLE
Trust Hierarchy & Entities

### DISCUSSION THREAD

**Discussion by Silverarmor** [GriefPrevention/GriefPrevention#857] [Ideas]
### Trust others to destroy entities, but not break blocks / access containers

I want to allow others to destroy **end crystal (entitiy)**, but not be able to break blocks or access containers. Is this possible?

### Hierarchy Question
Is it possible to allow a player to place/break blocks `/trust`, but not allow them to access containers `/containertrust`?

Thanks

**Comment by bigpresh** [comment]
No, it isn't; each "level" of trust (access trust -> container trust -> build trust) implies the previous levels.

I'm not sure if there are any plans to change this in GP v20.

If it was, there would be gotchas.  For instance, if you allow someone to place/break blocks, but not access your containers (chests etc) - what happens when they use their ability to break blocks to break the container, and all the contents are now dropped on the floor for them to pick up?
